src/components/PopularityIndicator.vue

```diff
@@ -1,18 +1,23 @@
 <script setup lang="ts">
-import { computed } from "vue"
+import { computed, ref } from "vue"
 import IndicatorBar from "./IndicatorBar.vue"
 
 const props = defineProps<{
   value: number
 }>()
 
-const areHappy = computed(() => props.value >= 50)
-const areAngry = computed(() => props.value < 50)
+const indicator = ref(props.value)
+const areHappy = computed(() => indicator.value >= 50)
+const areAngry = computed(() => indicator.value < 50)
+
+function changed(newValue: number) {
+  indicator.value = newValue
+}
 </script>
 
 <template>
   <div id="popularity">
-    <IndicatorBar id="indicator" title="Beliebtheit" :value="value" />
+    <IndicatorBar id="indicator" title="Beliebtheit" :value="indicator" @change="changed" />
```

Comment:
This seems to break the reactive connection with the state. What was your intention, here?

Reply:
I think it might help to use `toRef()` instead of `ref()`.

Reply:
Do you think of `const indicator = toRef(props, 'value')`? I don't understand the difference, can you pls explain?

Reply:
Yes, but I did not try it. If I understood it correctly, `ref()` takes the initial value as initialization for a fresh, independent reactive data, which is then changed independently - here by the click event. It does not get any update.

I think, this is the cause of the indicator bars to show wrong values at the start of the game: All three should be at 50%, but one is at 100% and two are at 0%.

Reply:
See below...